Browser support matrix - Added  browser support table to “Getting started / For engineers" (#1432)
hashicorp/design-system: website/docs/getting-started/for-engineers.md

```

For more examples and guidelines read [the tokens documentation](/foundations/tokens).

## Browser support

> The code of our styles, components and icons supports the following browsers:

Reviewer: It almost feels like the causality is the other way around – browsers support features rather than the other way around – but I can't think of better wording.

Author: @alex-ju what do you think of this alternative?
```
Our styles, components and icons are supported by the following browsers:
```

Reviewer: yep, that reads better in my opinion

> The code of our styles, components and icons supports the following browsers:

Author: ```suggestion
Our styles, components and icons are supported by the following browsers:
```
